site/content/docs/commands/task-run.en.md

```diff
@@ -63,7 +63,8 @@ Task Configuration Flags
       --platform-os string             Optional. Operating system of the task. Must be specified along with 'platform-arch'.
       --resource-tags stringToString   Optional. Labels with a key and value separated by commas.
                                        Allows you to categorize resources. (default [])
-      --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated by commas. (default [])
+      --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated by commas. (default []). 
+                                       For secrets stored in AWS Parameter Store you can either specify name or ARN. For the secrets stored in AWS Secrets Manager you need to specify ARN.
```

Comment:
to keep lines <80 width
```suggestion
                                       For secrets stored in AWS Parameter Store you can either specify name or ARN.
                                       For the secrets stored in AWS Secrets Manager you need to specify ARN.
```

Reply:
nit: should it be "...the names or the ARNs" and "you need to specify the ARNs"? @dannyrandall @huanjani @efekarakus english experts 🥺

Reply:
i think the plurals you suggested make more sense!